src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+public class Duke {
+
+    private static void printAddTaskConfirmation(Task task, int numTasks) {
+        String line = "____________________________________________________________";
+        System.out.println(line);
+        System.out.println("Got it. I've added this task:\n" + task + "\nNow you have " + numTasks + " tasks in the list.");
+        System.out.println(line);
+    }
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        //System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        
+        String line = "____________________________________________________________";
+
+        System.out.println(line);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        System.out.println(line);
+        
+        Task[] tasks = new Task[100];
+        int numTasks = 0;

```

Comment:
Is there any reason you put this here instead of making it a static class variable in Task?